h5p-bildetema/src/components/SearchPage/SearchPage.tsx

```diff
@@ -199,6 +188,9 @@ const SearchPage = (): JSX.Element => {
               searchLanguage={searchLanguage}
               handleSearchLanguageChange={handleSearchLanguageChange}
               searchInputPlaceholder={searchInputPlaceholder}
+              // Todo handle multiple languages
+              viewLanguage={viewLanguages.length > 0 ? viewLanguages[0] : null}
```

Comment:
Make a function called that uses something like  let updatedLanguages: Language[];  

```
  if (updatedLanguages.length === 0) {
    searchParams.delete("viewLang");
  } else {
    searchParams.set("viewLang", updatedLanguages.map(lang => lang.code).join(","));
  }
```

Then sets the langues with   
```
setSearchParams(searchParams);
setViewLanguages(updatedLanguages);
```

Reply:
Yes, we have waited with this one since we have been a bit back and forth about if the view language should be one or multiple languages. We have tested multiple before, but then we went back to only allowing users to select one language. Have kept the TODO comment as a reminder if they want to have multiple languages again. If they do, we'll add this in a new PR

Reply:
Alright I will mark it as resolved